pickle_server/exceptions.py

```diff
@@ -0,0 +1,72 @@
+import re
+from ninja.errors import AuthenticationError
+from django.core.exceptions import PermissionDenied, ObjectDoesNotExist
+from django.core.exceptions import ValidationError as DjangoValidationError
+from ninja.errors import ValidationError as NinjaValidationError
+
+from typing import Optional
+
+from ninja import Schema
+from ninja.responses import codes_2xx, codes_4xx
+
+
+class APIError(Exception):
+    status_code = 500
+    detail = "Internal Server Error"
+
+    def __init__(self, detail=None, status_code=None):
+        if detail:
+            self.detail = detail
+        if status_code is not None:
+            self.status_code = status_code
+
+    def __str__(self):
+        return self.detail
+
+
+class ResponseDTO(Schema):
```

Comment:
P3) 에러 응답 시에만 쓰이는 거죠? user views 와 네이밍 컨벤션 통일하면 어떨까요

```suggestion
class ErrorResponseSchema(Schema):
```

아니면 Dto로 통일해도 좋구요!

+) 크게 중요한 건 아니나, https://en.wikipedia.org/wiki/Camel_case#Programming_and_coding 여기 나와있는 예시들과 같은 엣지 케이스가 있기 때문에 두문자어의 경우에는 Xml Dto Html 이런 식으로 작성하는 걸 제안해봅니닷

Reply:
동의합니다 `ErrorResponseSchema`로 수정완료
두문자어 표기는 django만 해도 `HttpResponse`와 같이 표기하고있네요 보통 이렇게 쓰는거라는거 배워갑니다